src/libs/ajax/DataRepo.ts

```diff
@@ -104,7 +104,6 @@ export type AnySnapshotBuilderCriteria =
   | SnapshotBuilderProgramDataRangeCriteria
   | SnapshotBuilderProgramDataListCriteria;
 export interface SnapshotBuilderCriteriaGroup {
-  name: string;
```

Comment:
does this change to criteria group affect what is sent to the back end (maybe for the snapshot access request that contains criteria group)? If so, we may need to map it before it is sent so it matches the data structure on the back end or make a back end change so it matches

Reply:
Oh yes, this is a good reminder! Phil did mention to me something about opening a ticket to remove name in the backend repo as well